doc/contributing/contribute_code.md

```diff
@@ -10,19 +10,15 @@ project. For how to contribute to the documentation, see
 
 Make sure you have a Java SDK available to compile the application.
 
-The project has moved to the [Eclipse Temurin Java SDK](https://adoptium.net) 
+The project has moved to the [Apache Adoptium Java SDK](https://adoptium.net) 
```

Comment:
Are you absolutely sure that Temurin should be replaced by Adoptium?

Reply:
I've taken out this change